### PR TITLE
Migration of non-hacky changes from `feat/sampling_2024`

### PR DESCRIPTION
**Pull request by ChengshuLi**

### CODE REVIEW

File: omnigibson/systems/micro_particle_system.py [hunk 1]
```diff
@@ -1594,7 +1585,7 @@ def clothify_mesh_prim(cls, mesh_prim, remesh=True, particle_distance=None):
             particle_distance = cls.particle_contact_offset * 2 / 1.5 if particle_distance is None else particle_distance
 
             # Repetitively re-mesh at lower resolution until we have a mesh that has less than MAX_CLOTH_PARTICLES vertices
-            for _ in range(3):
+            for _ in range(10):
```
Comment (cremebrule) [inline omnigibson/systems/micro_particle_system.py:1588]: i think there should be the other changes also that allow for better remeshing, right?

**Review by cremebrule — APPROVED** [review]
lgtm, some minor comments

File: omnigibson/systems/macro_particle_system.py [hunk 2]
```diff
@@ -571,6 +574,7 @@ def generate_group_particles_on_object(cls, group, max_samples, min_samples_for_
                 aabb_offset=cls._SAMPLING_AABB_OFFSET,
                 max_sampling_attempts=cls._SAMPLING_MAX_ATTEMPTS,
                 refuse_downwards=True,
+                hit_proportion=cls._SAMPLING_HIT_PROPORTION,
```
[the remark this answers is not in the document]
Comment (ChengshuLi) [reply omnigibson/systems/macro_particle_system.py:577]: This is to reduce the hit proportion requirement for sampling *visual* particles. When we bump up the requirement, I think it's getting more difficult to sample visual particles (e.g. stain) on small objects (e.g. ring).

File: omnigibson/systems/micro_particle_system.py (hunk 1 above)
Comment (ChengshuLi) [reply omnigibson/systems/micro_particle_system.py:1588]: we can decide whether to include those changes into `og-develop` later

**Pull request merged**